lib/galaxy/datatypes/neo4j.py

```diff
@@ -99,36 +121,73 @@ def __init__(self, **kwd):
-        self.add_composite_file('neostore.propertystore.db.arrays', is_binary=True)
-        self.add_composite_file('neostore.propertystore.db.arrays.id', is_binary=True)
-        self.add_composite_file('neostore.propertystore.db.index', is_binary=True)
-        self.add_composite_file('neostore.propertystore.db.index.id', is_binary=True)
-        self.add_composite_file('neostore.propertystore.db.index.keys', is_binary=True)
-        self.add_composite_file('neostore.propertystore.db.index.keys.id', is_binary=True)
-        self.add_composite_file('neostore.propertystore.db.strings', is_binary=True)
-        self.add_composite_file('neostore.propertystore.db.strings.id', is_binary=True)
-
-        self.add_composite_file('neostore.relationshipgroupstore.db', is_binary=True)
-        self.add_composite_file('neostore.relationshipgroupstore.db.id', is_binary=True)
-        self.add_composite_file('neostore.relationshipstore.db', is_binary=True)
-        self.add_composite_file('neostore.relationshipstore.db.id', is_binary=True)
-        self.add_composite_file('neostore.relationshiptypestore.db.names', is_binary=True)
-        self.add_composite_file('neostore.relationshiptypestore.db.names.id', is_binary=True)
+        self.add_composite_file(
+            'neostore.propertystore.db.arrays', is_binary=True)
+        self.add_composite_file(
+            'neostore.propertystore.db.arrays.id', is_binary=True)
+        self.add_composite_file(
+            'neostore.propertystore.db.index', is_binary=True)
+        self.add_composite_file(
+            'neostore.propertystore.db.index.id', is_binary=True)
+        self.add_composite_file(
+            'neostore.propertystore.db.index.keys', is_binary=True)
+        self.add_composite_file(
+            'neostore.propertystore.db.index.keys.id', is_binary=True)
+        self.add_composite_file(
+            'neostore.propertystore.db.strings', is_binary=True)
+        self.add_composite_file(
+            'neostore.propertystore.db.strings.id', is_binary=True)
+
+        self.add_composite_file(
+            'neostore.relationshipgroupstore.db', is_binary=True)
+        self.add_composite_file(
+            'neostore.relationshipgroupstore.db.id', is_binary=True)
+        self.add_composite_file(
+            'neostore.relationshipstore.db', is_binary=True)
+        self.add_composite_file(
+            'neostore.relationshipstore.db.id', is_binary=True)
+        self.add_composite_file(
+            'neostore.relationshiptypestore.db.names', is_binary=True)
+        self.add_composite_file(
+            'neostore.relationshiptypestore.db.names.id', is_binary=True)
         self.add_composite_file('neostore.schemastore.db', is_binary=True)
         self.add_composite_file('neostore.schemastore.db.id', is_binary=True)
         self.add_composite_file('neostore.transaction.db.0', is_binary=True)
 
 
+class Neo4jDBzip(Neo4j, Data):
```

Comment:
For this version of there is no reason for your recommendation to be implemented. It will help the user in the upload process basically.